### ISSUE TITLE
allowing aria-labelledby in img

### ISSUE THREAD

**Issue by donnanicolas** [evcohen/eslint-plugin-jsx-a11y#411]
The current rules for `alt-text` don't allow images with `aria-labelledby` to have no `alt`. But it seems that the w3 has documentation regarding this technique: https://www.w3.org/WAI/GL/wiki/Using_aria-labelledby_to_provide_a_text_alternative_for_non-text_content

The use case I'm referring to is when I have an image an immediately below it, I have the text for that image, wouldn't that make an screen reader read the same text twice??

**Comment by donnanicolas** [comment]
Any news regarding this??

**Comment by dimaqq** [comment]
I'm curious about something similar to this, would `<img aria-label="Close" src=.../>` (without alt) be correct?
Today the plugin doesn't like this style.
Likewise for `<img title="Close" src=.../>`.
Perhaps I misunderstand the semantics of aria-label/title/alt ?

**Comment by jessebeach** [comment]
> I'm curious about something similar to this, would `<img aria-label="Close" src=.../>` (without alt) be correct?
> Today the plugin doesn't like this style.
> Likewise for `<img title="Close" src=.../>`.
> Perhaps I misunderstand the semantics of aria-label/title/alt ?

@dimaqq you can test the AX Tree output for the variations in Chrome's Accessibility Panel in the DOM Inspector. I think they'll all produce the correct AX Tree output, so functionally they should be equivalent and there's some improvement to be made in this rule's behavior.
